scripts/update-docker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Used to restart a single msgr api container running on a server
+# First kills all running containers named msgr_api then starts a new container with the name msgr_api
+
+if [ ! -z $(docker ps -q --filter name="msgr_api") ] 
+then
+    echo "Killing all running docker processes"
+    docker kill $(docker ps -q)
+fi
+
+echo "Running new process"
+
+docker run --rm -t -d --name "msgr_api" -p 80:5000 -v /home/ec2-user/cs340_final_project/api/flask_app/configuration.py:/app/configuration.py jemisonf/msgr_api:latest
```

Comment:
Good idea on cleaning up the dead container by using the `--rm` option. I'm wondering if we should handle it this way though rather than adding a remove step after the kill step above. If the container goes down for some reason, we wouldn't be able to view the logs for what caused it if it immediately was removed after exiting. What are your thoughts on this?

Reply:
Both this and -t can probably be removed, they're both basically holdovers from the container run command I usually use